src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/TestPtvConsts.java

```diff
@@ -0,0 +1,78 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.ptv;
+
+public class TestPtvConsts {
+  
+  public static final String[] ORGANIZATIONS = {
+    "ae2682d3-6238-4019-b34f-b078c5f9bb50",
+    "f3cb50b9-e97f-4c00-a9c7-04265bdcadb6",
+    "9355a207-efd3-4cfb-a02b-67187f34c822"
+  };
+
+  public static final String[] SERVICES = {
+    "2f21448e-e461-4ad0-a87a-47bcb08e578e",
+    "bcfaf5f1-98db-401e-8628-7d4f2b4ec260",
+    "4c490d84-1fc2-4011-98b5-fbc7a878cbce",
+    "b21dbc0e-ea54-449d-8419-3fe7a6126948",
+    "6b979930-048e-4ed4-a80c-0e11ba562dfe",
+    "58a33fee-d7c6-4875-a1b7-75d70b00d012",
+    "08ab5b97-d04d-4d16-a3f3-dac654b140ec",
+    "2fb268a5-8fa5-467e-b908-98131b916365",
+    "8f0fe030-794e-4ed5-beb7-57c6be43f2ba"
+  };
+
+  public static final String[] SERVICE_CHANNELS = {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2386)